### PR TITLE
Add hms topic

### PR DESCRIPTION
**Pull request by Mike-mei**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of why you think this topic or collection should be curated.

The #hms is a commonly used topic and has been used in 61 repositories. More and more developers will use this topic in the future, so this topic need to be curated.

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by Mike-mei** [comment]
> Can you remove the changes to the PR template?

I have deleted the changes to the PR template

**Comment by kenyonj** [comment]
It looks like you deleted the template all together. Can you add it back in, without any changes? Thanks!

**Comment by Mike-mei** [comment]
> It looks like you deleted the template all together. Can you add it back in, without any changes? Thanks!
How can I add it back in? I didn't find the button to recover deleted files.

**Comment by kenyonj** [comment]
> > It looks like you deleted the template all together. Can you add it back in, without any changes? Thanks!
> > How can I add it back in? I didn't find the button to recover deleted files.

It looks like you'll have to either do this in the command line by checking out the file from the `master` branch, or, you can add another file to this PR and just copy and paste the contents of the file from `master`.

**Comment by Mike-mei** [comment]
> > > It looks like you deleted the template all together. Can you add it back in, without any changes? Thanks!
> > > How can I add it back in? I didn't find the button to recover deleted files.
> 
> It looks like you'll have to either do this in the command line by checking out the file from the `master` branch, or, you can add another file to this PR and just copy and paste the contents of the file from `master`.

I add pull request template back in

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
+
+- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
+- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
+- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs
+
+> Please replace this line with an explanation of why you think this topic or collection should be curated.
+
+---------------------------------------------------------------------
+
+**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**
```
Comment (kenyonj) [inline .github/PULL_REQUEST_TEMPLATE.md:33]: Looks like this is still different than what is on master. I believe if you accept this suggestion is should fix it up.

```suggestion
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [ ] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [ ] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

<!-- ⚠️ Please select either this section... ⚠️ -->
### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

<!-- ⚠️ ... or this section ⚠️ -->
### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of why you think this topic or collection should be curated.

<!-- ⚠️ ... or this section ⚠️ -->
### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**
```

**Comment by Mike-mei** [comment]
> OK, now that we're back to valid test failures, here is one that needs attention:
> 
> ```shell
> topics::hms topic#test_0024_has a valid short_description [/home/runner/work/explore/explore/test/topics_test.rb:291]:
> must have a short_description no more than 130 characters (currently 152)
> ```

I have changed the short description to no more tha 130 characters

**Comment by kenyonj** [comment]
Thanks for your work on this @Mike-mei !

**Pull request merged**